code/game/objects/items/weapons/tools.dm

```diff
@@ -396,25 +405,57 @@
 /obj/item/weapon/weldingtool/hugetank
 	name = "upgraded welding tool"
 	max_fuel = 80
-	w_class = 3.0
+	w_class = 2.0
 	matter = list(DEFAULT_WALL_MATERIAL = 70, "glass" = 120)
 	origin_tech = "engineering=3"
 
+
+
+
+//The Experimental Welding Tool!
 /obj/item/weapon/weldingtool/experimental
 	name = "experimental welding tool"
+	desc = "A scientifically-enhanced welding tool that uses fuel-producing microbes to gradually replenish its fuel supply"
 	max_fuel = 40
-	w_class = 3.0
+	w_class = 2.0
 	matter = list(DEFAULT_WALL_MATERIAL = 70, "glass" = 120)
-	origin_tech = "engineering=4;phorontech=3"
+	origin_tech = "engineering=4;biotech=4"

```

Comment:
For reference. Let's make this into like, `"engineering=4;materials=2"` or something like that.